tests/Testing/Endpoints.cs

```diff
@@ -15,10 +15,12 @@ public static string GetUpdated(string[]? bcp = null)
                 bcpParam = string.Join("&", bcp.Select(x => $"bcp={x}")) + "&";
             }
 
-            return $"{Root}?{bcpParam}from=2024-11-20";
+            return $"{Root}?{bcpParam}from={IsoDate(from)}&to={IsoDate(to)}";
         }
 
         public static string Get(string chedReferenceNumber = ChedReferenceNumbers.ChedA) =>
             $"{Root}/{chedReferenceNumber}";
+
+        private static string IsoDate(DateTime date) => date.ToString("yyyy-MM-ddTHH:mm");
```

Comment:
ISO 8601 can be formatted using the short hand of `ToString("O")` rather than a custom pattern